### DISCUSSION TITLE
添加新的zotero插件——zotero box

### DISCUSSION THREAD

**Discussion by 11good11** [syt2/zotero-addons#95] [Ideas]
https://gitee.com/scigreat/zotbox，麻烦作者将这个插件添加进插件市场，这也是一个不错的插件

**Comment by syt2** [comment]
`addon-scraper` 插件源临时增加了该插件  
对于主数据源[zotero-chinese](https://github.com/zotero-chinese/zotero-plugins)，可能需要等待 https://github.com/zotero-chinese/zotero-plugins/issues/123 的feature开发完成之后才能支持该插件
